src/System.Data.Common/src/System/Data/IDataParameter.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Data {
```

Comment:
Nit: the coding style for corefx is Allman style braces, where the braces start on a new line.